tests/test_pe.py

```diff
@@ -17,6 +17,12 @@
 from unicorn.x86_const import *
 
 class PETest(unittest.TestCase):
+
+    def test_pe_dos_8086_hello(self):
+        ql = Qiling(["../examples/rootfs/8086_dos/HI.COM"], "../examples/rootfs/8086_dos")
```

Comment:
I suggest we  can make it test_dos.py so we can run the same test in mac and linux

Reply:
See commit 2cc0303.